<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTextWidget</class>
 <widget class="QWidget" name="SearchTextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>28</height>
   </rect>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="BaseLineEdit" name="m_txtSearch">
     <property name="placeholderText">
      <string>Search text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnSearchBackward">
     <property name="toolTip">
      <string>Find previous occurence</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnSearchForward">
     <property name="toolTip">
      <string>Find next occurence</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseLineEdit</class>
   <extends>QLineEdit</extends>
   <header>baselineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_txtSearch</tabstop>
  <tabstop>m_btnSearchBackward</tabstop>
  <tabstop>m_btnSearchForward</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
